### ISSUE TITLE
Add support for importing VPC route table routes

### ISSUE THREAD

**Issue by YakDriver** [terraform-providers/terraform-provider-aws#5689]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Allow `aws_route` resources to be imported into the current state.

### New or Affected Resource(s)

* aws_route

### Potential Terraform Configuration

```hcl
resource "aws_route" "internal-default-route" {
  route_table_id = "${aws_route_table.client-routes.id}"
  destination_cidr_block = "0.0.0.0/0"
  network_interface_id = "${aws_network_interface.router-internal.id}"
}
```

To import the route:
```console
$ terraform import aws_route.my_route r-rtb-656C65616E6F72_0.0.0.0/0
```

### References

* #704
* #5687
* #5631

**Comment by bflad** [comment]
Support for importing `aws_route` resources has been merged into master and will release with version 1.34.0 of the AWS provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 1.34.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1340-august-30-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
